include/rc_consoles.h

```diff
@@ -94,6 +94,7 @@ enum {
   RC_CONSOLE_TI83 = 79,
   RC_CONSOLE_UZEBOX = 80,
   RC_CONSOLE_FAMICOM_DISK_SYSTEM = 81,
+  RC_CONSOLE_WIIWARE = 82,
```

Comment:
I don't think this deserves a separate console id. WiiWare is just a different way to distribute games for Wii, just like [DSIWare](https://retroachievements.org/hub/20698) is for DSi.

Or disk vs cartridge for something like the Commodore 64 or MSX.